main/cmake/APKBUILD

```diff
@@ -41,6 +41,7 @@ build() {
 		--system-curl \
 		--system-expat \
 		--no-system-jsoncpp \
+		--no-system-librhash \
```

Comment:
Yep, will undo the change removing `--no-system-jsoncpp` and wait for `rhash` to be fixed.